app/models/cms/group.rb

```diff
@@ -4,7 +4,6 @@ class CMS::Group < ApplicationRecord
   # This is BuyerGroup
   self.table_name = :cms_groups
 
-  belongs_to :tenant
```

Comment:
see description of  30f1e11dfa96

Reply:
I don't know what `CMS::Group` is. I would say it's unused. I worked on the CMS API and it doesn't support this class at all.

Reply:
probably, forums need to be cleaned, we had a JIRA issue somewhere